refinery/analysis_manager/models.py

```diff
@@ -43,15 +43,15 @@ class AnalysisStatus(models.Model):
     class Meta:
         verbose_name_plural = 'analysis statuses'
 
-    def __unicode__(self):
+    def __str__(self):
         return self.analysis.name
 
     def set_galaxy_history_state(self, state):
         """
         Set the `galaxy_history_state` of an analysis
         :param state: a valid GALAXY_HISTORY_STATE
         """
-        if state in dict(self.GALAXY_HISTORY_STATES).keys():
+        if state in list(dict(self.GALAXY_HISTORY_STATES).keys()):
```

Comment:
`list()` should not be necessary since `keys()` returns an iterable

Reply:
i am aware - this was from 2to3, whose changes i did not touch.  i can update these if you'd prefer though